pkg/core/core.go

```diff
@@ -259,20 +251,20 @@ func readChaosBountyPrograms() (map[string]chaosProgram, error) {
 	}
 	defer file.Close()
 
-	var list chaosList
+	var list ChaosList
 	if err := json.NewDecoder(file).Decode(&list); err != nil {
 		return nil, errors.Wrap(err, "could not decode chaos list")
 	}
 
-	chaosMap := make(map[string]chaosProgram)
+	chaosMap := make(map[string]ChaosProgram)
 	for _, value := range list.Programs {
 		chaosMap[value.Name] = value
 	}
 	log.Printf("[INFO] Read %d programs from chaos list\n", len(chaosMap))
 	return chaosMap, nil
 }
 
-func extractHostname(item string) string {
+func ExtractHostname(item string) string {
```

Comment:
This function does more than just validating if `item` is a valid domain (eg. converts to lowercase and strip prefixes). I'd suggest to move the anonymous function `validate` into a package exported function (eg. `ValidateFQDN`), since it contains the logic that we need.